site/content/contribute/more-info/mobile/developer-setup/_index.md

```diff
@@ -225,8 +268,8 @@ The following build commands failed:
 
 Check the following things:
 
-* Ensure you are running the latest version of `nvm` using the {{< newtabref href="https://github.com/nvm-sh/nvm#install--update-script" title="Upgrade Instructions" >}}
+* Ensure you are running the latest version of `nvm` using the {{< newtabref href="https://github.com/nvm-sh/nvm#install--update-script" title="upgrade instructions" >}}.
 * Ensure you have set your desired version of node in the file `~/.nvmrc`.  E.g.,
-    ```sh
-      echo v16.2.0 > ~/.nvmrc
-    ```
+  ```sh
+  echo v16.2.0 > ~/.nvmrc
+  ```
```

Comment:
I'm not sure this is needed, but in case we want to keep it the version should be v18+

Also the section of `PhaseScriptExecution` can be removed